recipes/solr-deploy.rb

```diff
@@ -276,6 +276,8 @@
         # If not, then it's either a directory (obsolete) or malformed, so ignore it.
         if (tar tzf "$LATEST_INDEX" >/dev/null 2>&1); then
             mkdir -p "$CORE_DATA/index"
+            # remove the index.properties file so default index config is used
+            rm -f $CORE_DATA/index.properties
```

Comment:
just checking, this is only run on 'new' instances not if chef is run a second time?

have we dont BCP/DR testing to see that it heals from 0 nodes to 2 nodes in dev yet with this fix?

Reply:
I believe it would repeat on each deployment. Which shouldn't be a problem, because Solr doesn't actually need the file, at least in our setup.